homeassistant/components/wallbox/config_flow.py

```diff
@@ -34,6 +34,31 @@ async def validate_input(hass: core.HomeAssistant, data):
 class ConfigFlow(config_entries.ConfigFlow, domain=COMPONENT_DOMAIN):
     """Handle a config flow for Wallbox."""
 
+    def __init__(self):
+        """Start the Wallbox config flow."""
+        self._reauth_entry = None
+        self._username = None
+        self._station = None
+        self.reauth_schema = None
+
+    async def async_step_reauth(self, user_input=None):
+        """Perform reauth upon an API authentication error."""
+        self._reauth_entry = self.hass.config_entries.async_get_entry(
+            self.context["entry_id"]
+        )
+
+        self.reauth_schema = {
+            vol.Required(
+                CONF_STATION, default=self._reauth_entry.data[CONF_STATION]
+            ): str,
+            vol.Required(
+                CONF_USERNAME, default=self._reauth_entry.data[CONF_USERNAME]
+            ): str,
+            vol.Required(CONF_PASSWORD): str,
+        }
+
+        return await self.async_step_user()
```

Comment:
This is a different feature. Please split different features into separate PRs.

Thanks 👍

Reply:
Yes, I might have been a bit overambitious, adding it all in one PR ;-). I'll move the reauth part to a seperate / next PR